department-of-veterans-affairs/caseflow
# Add CAMO in progress tab

Resolves [CASEFLOW-2592](https://vajira.max.gov/browse/CASEFLOW-2592)

### Description
Adds an "In Progress" tab to CAMO queue to show cases that are with lower VHA orgs. Excludes cases that are "on hold" with those lower orgs.

### Acceptance Criteria

### Testing Plan
1. Ensure PO/VISN level assigned and in progress tabs show in new tab
2. Ensure PO/VISN level on hold tasks don't show in new tab
